gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -884,16 +923,17 @@ accept_socket_async_cb (GObject * source, GAsyncResult * result,
 }
 
 /**
- * @brief set server source and sink tensors config.
+ * @brief set server source and sink caps string.
  */
 void
-nnstreamer_query_server_data_set_config (query_server_handle server_data,
-    GstTensorsConfig * src_config, GstTensorsConfig * sink_config)
+nnstreamer_query_server_data_set_caps_str (query_server_handle server_data,
+    const char *src_caps_str, const char *sink_caps_str)
 {
   TensorQueryServerData *sdata = (TensorQueryServerData *) server_data;
-
-  gst_tensors_config_copy (&sdata->src_config, src_config);
-  gst_tensors_config_copy (&sdata->sink_config, sink_config);
+  g_free (sdata->src_caps_str);
+  g_free (sdata->sink_caps_str);
+  sdata->src_caps_str = g_strdup (src_caps_str);
```

Comment:
need free function in nnstreamer_query_server_data_free()
```
void
nnstreamer_query_server_data_free (query_server_handle server_data)
  .....
  g_free (sdata->src_caps_str);
  g_free (sdata->sink_caps_str);
  g_free (sdata);
```

Reply:
Thanks, Updated.